webapp/components/Modal/ConfirmModal.spec.js

```diff
@@ -50,7 +50,9 @@ describe('ConfirmModal.vue', () => {
       beforeEach(() => {
         wrapper = Wrapper()
       })
-
+      it('isOpen true', () => {
+        expect(wrapper.vm.isOpen).toBe(true)
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-21T12:53:52Z" title="Friday, June 21st 2019, 2:53:52 pm +02:00">Jun 21, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -50,7 +50,9 @@ describe('ConfirmModal.vue', () => {
       beforeEach(() => {
         wrapper = Wrapper()
       })
-
+      it('isOpen true', () => {
+        expect(wrapper.vm.isOpen).toBe(true)
```


Try to avoid directly accessing a components internal state. If there is a way to check if sth. is visible, that would be a much nicer way of doing it.